docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -0,0 +1,207 @@
+[[Galleon_Provisioning]]
+= Provisioning WildFly with Galleon
+
+WildFly server being internally built using the https://github.com/wildfly/galleon[Galleon project], WildFly benefits +
+from Galleon provisioning capabilities. One of the main feature that Galleon brings to WildFly is the ability
+to install complete or customized WildFly server using Galleon tooling.
+
+More information on Galleon features and tooling can be found on https://docs.wildfly.org/galleon/[this page].
+
+== WildFly Galleon feature-packs
+
+WildFly is packaged as a Galleon feature-pack maven artifact (zipped file that contains installation content). 
+This feature-pack, as well as the feature-packs on which its depends (core and servlet feature-packs),
+are deployed in public maven repositories. 
+
+When using Galleon to install WildFly, WildFly feature-packs are retrieved and 
+their content is assembled to create an installation.
+
+== Installing WildFly using Galleon
+
+Galleon maven plugin or Galleon CLI are used to install WildFly. Latest CLI and documentation can be downloaded 
+from https://github.com/wildfly/galleon/releases[this page].
+
+To install the latest final version of WildFly into the directory _my-wildfly-server_ call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server
+----
+
+Once installed, the directory _my-wildfly-server_ contains all that is expected to run a complete WildFly server. 
+By default, all standalone and domain configurations are installed.
+
+NB:
```

Comment:
Does an asciidoc "Admonition" work for the various NB items you have?

https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#more-delimited-blocks

Reply:
I added proper NOTE.